repos/re-theme/src/hooks/index.js

```diff
@@ -3,3 +3,4 @@ export * from './useStylesCallback'
 export * from './useThemeHover'
 export * from './useThemeActive'
 export * from './useThemeFocus'
+export * from './useStylesMemo'
```

Comment:
This seems to be a duplicate of the `useStyle` hook is keg-components.

Reply:
spoke on slack -- updated to delete the `keg-components` version and renamed this hook to be `useStyle`